reference/auth.v1.yaml

```diff
@@ -722,6 +722,24 @@ paths:
       security:
         - serverToken: []
 
+  '/auth/v1/users/{userId}/device_tokens':
+    parameters:
+      - $ref: './common/parameters/tidepooluserid.yaml'
+    post:
+      operationId: PostDeviceToken
```

Comment:
Consider renaming this to `CreateDeviceTokenForUser` or something more descriptive. The operation id is used to generate client interface.